### PR TITLE
Feature/hypervisor new app flags

### LINKED ISSUE #609: Add new flags to hypervisor app management
**Feature description**

Two new flags will be added to the VPN with the next release. 

- killswitch
- secure

Both accept boolean values. They need to be changeable via the app management interface in the UI. 

Reference PRs

- https://github.com/skycoin/skywire/pull/608/
- https://github.com/skycoin/skywire/pull/598

### PR DESCRIPTION
**Pull request by i-hate-nicknames**
Did you run `make format && make check`? Yes

Fixes #609

 Changes:	
- Add secure and killswitch flags to hypervisor api

How to test this PR:
setup an env with vpn client and server and try adding new flags with [updated frontend](https://github.com/skycoin/skywire/pull/617)

### CODE REVIEW

File: pkg/visor/api.go [hunk 1]
```diff
@@ -303,6 +305,72 @@ func (v *Visor) SetAppPassword(appName, password string) error {
...
+// SetAppKillswitch implements API.
+func (v *Visor) SetAppKillswitch(appName string, killswitch bool) error {
+	if appName != skyenv.VPNClientName {
+		return fmt.Errorf("app %s is not allowed to set killswitch", appName)
+	}
+
+	v.log.Infof("Setting %s killswitch to %q", appName, killswitch)
+
+	const (
+		killSwitchArg = "-killswitch"
```
Comment (taras-skycoin) [inline pkg/visor/api.go:317]: Are you sure this is correct format? @Darkren wrote in chat that I need to use it like this `--killswitch=true`

**Comment by taras-skycoin** [comment]
```
curl -k -d '{"killswitch":true}' -H "Content-Type: application/json" -X PUT http://localhost:8000/api/visors/023d1dc11119a7e2d2207f6c9e94ee88a5801702616ca1ff72bc3b2af785b7c1b8/apps/vpn-client
{"name":"vpn-client","args":["-srv","02a7643d3c4e493fcbbbd65ae75991db97a0c43345920b428ffe49a908daeb019b","-killswitch","true"],"auto_start":true,"port":43,"status":0}
```
Works for me

File: pkg/visor/api.go (hunk 1 above)
Comment (i-hate-nicknames) [reply pkg/visor/api.go:317]: The vpn-client/vpn server work with either version: single or double dashed.

The reason I used single dash is for consistency with other arguments like passcode that are also declared with a single dash in the code around there.

I did some reading, and it seems that it actually shouldn't work. Cobra maintains a fork of flag package, that handles single dash [differently](https://github.com/spf13/pflag#command-line-flag-syntax)

However, when I build vpn-client and vpn-server, they accept both single and double dash flags, like [stdlib flag](https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax).
The help message in vpn-server/vpn client actually suggests using single dash flags.

Maybe cobra is configured somehow to use stdlib behavior of flags, I'm not sure. Maybe @Darkren can comment on that.

Comment (Darkren) [reply pkg/visor/api.go:317]: @i-hate-nicknames  is actually right. I tried it the first time with `-killswitch` and it didn't work for me, probably I just messed something up back then. just tried it again, it works

File: pkg/visor/hypervisor.go [hunk 2]
```diff
@@ -553,6 +555,23 @@ func (hv *Hypervisor) putApp() http.HandlerFunc {
 			}
 		}
 
+		// todo: this was just a quick copypasting to mimic the style already present here
+		// possible issue: each cal to ctx.API.Set* triggers app restart, so if we change multiple
+		// values in a single request the app will be restarted multple times
```
Comment (Darkren) [inline pkg/visor/hypervisor.go:560]: That's actually true, probably is worth making a ticket @jdknives

**Review by Darkren — APPROVED** [review]
Great job!

File: pkg/visor/hypervisor.go (hunk 2 above)
Comment (jdknives) [reply pkg/visor/hypervisor.go:560]: @Darkren sure. Will do. Can be merged in the meantime in any case.

**Pull request merged**